rtos/TARGET_CORTEX/rtx5/rtx_lib.c

```diff
@@ -128,7 +128,12 @@ static const osThreadAttr_t os_idle_thread_attr = {
   &os_idle_thread_stack,
   (uint32_t)sizeof(os_idle_thread_stack),
   osPriorityIdle,
-  0U, 0U
+#if (__DOMAIN_NS == 1U)
+  1U,
```

Comment:
Actual fix should come in CMSIS, raised an issue https://github.com/ARM-software/CMSIS_5/issues/252.
Need to update when we have proper fix from CMSIS